docs/maps/maps-getting-started.asciidoc

```diff
@@ -1,129 +1,123 @@
 ++++
-<titleabbrev>Get started</titleabbrev>
+<titleabbrev>Create a map</titleabbrev>
 ++++
 
+If you are new to creating maps in {kib}, this tutorial is a good place to start.
+It guides you through the most common steps for plotting location data.
 
+You'll learn to:
 
-You work with Maps by adding layers. The data for a layer can come from
-sources such as {es} documents, vector sources, tile map services, web map
-services, and more. You can symbolize the data in different ways.
-For example, you might show which airports have the longest flight
-delays by using circles from small to big. Or,
-you might show the amount of web log traffic by shading countries from
-light to dark.
+- Create a map with multiple layers and data sources
+- Use symbols, colors, and labels to style data values
+- Focus in on just the data you want
+- Embed a map in a dashboard
 
-[role="screenshot"]
-image::maps/images/sample_data_web_logs.png[]
-
-[[maps-read-only-access]]
-NOTE: If you have insufficient privileges to create or save maps, a read-only icon
-appears in the application header. The buttons to create new maps or edit
-existing maps won't be visible. For more information on granting access to
-Kibana see <<xpack-security-authorization>>.
+When you complete this tutorial, you’ll have a map that looks like this:
 
 [role="screenshot"]
-image::maps/images/read-only-badge.png[Example of Maps' read only access indicator in Kibana's header]
+image::maps/images/sample_data_web_logs.png[]
 
 [float]
 === Prerequisites
-Before you start this tutorial, <<add-sample-data, add the web logs sample data set>>. Each
-sample data set includes a map to go along with the data. Once you've added the data, open Maps and
-explore the different layers of the *[Logs] Total Requests and Bytes* map.
-You'll re-create this map in this tutorial.
 
-[float]
-=== Take-away skills
-In this tutorial, you'll learn to:
+- If you don’t already have {kib}, set it up with https://www.elastic.co/cloud/elasticsearch-service/signup?baymax=docs-body&elektra=docs[our free trial].
+- This tutorial uses the <<get-started, web logs sample data set>>. The sample data includes a [Logs] Total Requests and Bytes map, which you’ll create in this tutorial.
+- You must have the correct privileges for creating a map. If you have insufficient privileges to create or save maps,
+a read-only icon appears in the application header. For more information on granting access to {kib},
+ see <<xpack-security-authorization,Granting access to {kib}>>.
 
-* Create a multi-layer map
-* Connect a layer to a data source
-* Use symbols, colors, and labels to style a layer
-* Create layers for {es} data
-
-[role="xpack"]
+[float]
 [[maps-create]]
-=== Create a map
-
-The first thing to do is to create a new map.
+=== Step 1. Create a map
 
-. If you haven't already, open the main menu, then click *Maps*.
-. On the maps list page, click *Create map*.
+. Open the main menu, then click *Dashboard*.
+. Click **Create new**.
```

Comment:
Add a step: "Click *Create dashboard*"
Also, how about moving the "Set the time range to *Last 7 days*." ahead of "Click **Create new**." so users are setting the time in dashboard.